pkg/controller/idler/idler_controller_test.go

```diff
@@ -102,7 +106,11 @@ func TestEnsureIdling(t *testing.T) {
 
 		// then
 		require.NoError(t, err)
-		assert.Equal(t, reconcile.Result{}, res)
+		// no need to requeue, the controller will reconcile when a pod is created
```

Comment:
This comment is wrong I guess
```suggestion
		// no pods found - the controller will requeue after the idler's timeout
```

Reply:
ah right, good catch! thanks @MatousJobanek!